ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -593,9 +593,8 @@ presubmits:
     context: ci/prow/e2e-libvirt
     decorate: true
     name: pull-ci-openshift-installer-master-e2e-libvirt
-    optional: true
     rerun_command: /test e2e-libvirt
-    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
+    run_if_changed: ^(bin|cmd|data/data/(bootstrap|manifests/openshift)|hack|images|openshift|(pkg/(asset/([a-zA-Z]+\.go|cluster/[a-zA-Z]+\.go|installconfig/[a-zA-Z]+\.go|ignition|installconfig/[a-zA-Z]+\.go|kubeconfig|machines/(machineconfig|[a-zA-Z]+\.go)|manifests/[a-zA-Z]+\.go|mock|password|rhcos|store|targets|templates|tls))|(destroy/([a-zA-Z]+\.go|providers))|gather|ipnet|lineprinter|rhcos|(terraform/([a-zA-Z]+\.go|exec))|(types/([a-zA-Z]+\.go|conversion|defaults|validation))|validate|version))$|libvirt
```

Comment:
That is *not* was we in the end agreed upon. The agreement was that this will be run for all changes that touch the generic and libvirt code. Which makes perfect sense for not just libvirt but other platforms too. Please talk to @brenton.

Reply:
@abhinavdahiya I hope you had a chance to discuss this with @brenton and have reconsidered your position on this?

Reply:
Just for the record, i had a chat with @abhinavdahiya about this. We agreed to go with his suggestion to only run this on libvirt changes. Eventually it's promised that CI will be randomly picking platforms to test so that will make up for lack of tests run for libvirt platform on changes to generic code.

With that in mind, i guess we should simply close this PR since AFAIK there is no way to make it mandatory for libvirt only and run it as optional for other platforms. IMO it's important that at least `libvirt-e2e` is run for all changes than having it as mandatory for libvirt-only changes. I hope the plan mentioned in the last para, is implemented soon.